server/data/quotes.json

```diff
@@ -204,11 +204,23 @@
     "author": " Tomáš Garrigue Masaryk"
   },
  {
-     "sentence": "Language is not a genetic gift, it is a social gift. Learning a new language is becoming a member of the club – the community of speakers of that language.",
-     "author": " Frank Smith"
+    "sentence": "Language is not a genetic gift, it is a social gift. Learning a new language is becoming a member of the club – the community of speakers of that language.",
+    "author": " Frank Smith"
+ },
+ {
+    "sentence": "A special kind of beauty exists which is born in language, of language, and for language.",
+    "author": "Gaston Bachelard"
  },
  {
-     "sentence": "A special kind of beauty exists which is born in language, of language, and for language.",
-     "author": "Gaston Bachelard"
+    "sentence": "One should not aim at being possible to understand but at being impossible to misunderstand.",
+    "author": "Marcus Fabius Quintilian"
+ },
+ {
+    "sentence": "With languages, you are at home anywhere.",
+    "author": "Edmund De Waal"
+ },
+ {
+    "sentence": "The limits of my language are the limits of my world.",
```

Comment:
This is a duplicate, please see lines `110-114`. Hope you can remove it before I merge :)

Reply:
yes I changed it. Let me know if it was deleted :)

Reply:
Great thanks!